client/container_wait.go

```diff
@@ -1,14 +1,19 @@
 package client // import "github.com/docker/docker/client"
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
+	"errors"
+	"io"
 	"net/url"
 
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/api/types/versions"
 )
 
+const containerWaitErrorMsgLimit = 2 * 1000 /* Max: 2K */
```

Comment:
```suggestion
const containerWaitErrorMsgLimit = 2 * 1024 /* Max: 2KiB */
```
Small nit, but kibibytes is a bit more standard here.